src/Chapter2.hs

```diff
@@ -842,7 +867,9 @@ list.
 
 🕯 HINT: Use the 'cycle' function
 -}
-rotate = error "rotate: Not implemented!"
+rotate n xs = if (n /= (abs n)) then
```

Comment:
Is this a check on negative number?
In that case, you can be more explicit about it, I think 🙂